build.sbt

```diff
@@ -36,7 +36,7 @@ val zioPreludeVersion         = "1.0.0-RC19"
 
 inThisBuild(
   List(
-    scalaVersion             := scala212,
+    scalaVersion             := scala212, // Change to control IntelliJ's highlighting
```

Comment:
When this is changed to `scala213` / `scala3`, intelliJ should pick the right configuration and do highlighting properly